File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = ''
    
    # 1)
    # if int(str(a) + str(b)) > int(str(b) + str(a)):
    #     answer = int(str(a) + str(b))
    # else:
    #    answer = int(str(b) + str(a))
    
    # 2)
    answer = int(max(f"{a}{b}",f"{b}{a}"))
    
    return answer

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int a, int b) {
    int answer = 0;

    int ab = cal(a,b);
    int ba = cal(b,a);
    
    if(ab >= ba){
        answer = ab;
    }else {
        answer = ba;
    }
    
    printf("%d,%d",ab,ba);

    return answer;
}

int cal(int e1, int e2) {   
    int len = 0;
    int temp = e2;
    
    while(temp > 0) {
        temp /= 10;
        len++;
    }

    return e1 * (int) pow(10, len) + e2;
}